### PR TITLE
[REF] website_product_filters: display only one level of subcategories on categories column HU#1006 CA#27

### PR DESCRIPTION
**Pull request by oscarolar**
This PR is part of this whole issue:  https://github.com/Vauxoo/yoytec/issues/489
Build https://github.com/Vauxoo/yoytec/pull/631
- [x] After mergeing this merge this too: https://github.com/Vauxoo/SerpentCS_Contributions-v8/pull/20

### CODE REVIEW

File: website_product_filters/models/product.py [hunk 1]
```diff
@@ -80,11 +80,10 @@ class ProductCategory(models.Model):
         "product_template_id", readonly=True)
     total_tree_products = fields.Integer("Total Subcategory Prods",
                                          compute="_get_product_count",
-                                         store=True,)
+                                         store=False,)
```
Comment (oscarolar) [inline website_product_filters/models/product.py:83]: @moylop260 dude, this is ok for you?

Comment (moylop260) [inline website_product_filters/models/product.py:83]: @oscarolar 
I don't have a full context of this change, but the `store` param affect directly the instance migrated, you should ask to @josemoralesp

Comment (oscarolar) [inline website_product_filters/models/product.py:83]: @josemoralesp I made this change, when I coded this it was intended to be `store=False` do you have any problem with this?

**Comment by oscarolar** [comment]
This is what has been achieved so far in order to solve https://github.com/Vauxoo/yoytec/issues/489:
iamge 

@nhomar any comments are welcome

**Comment by oscarolar** [comment]
So far with no external dependencies installed (yoytec in this case) this is how the App works now: 
NOE

File: website_product_filters/models/product.py (hunk 1 above)
Comment (nhomar) [inline website_product_filters/models/product.py:83]: @oscarolar I prefere store = True with a good trigger, but I do not know if it is technically possible, compute this field any time view is loaded is too time consuming.

Can you propose a solution for that?

File: website_product_filters/controllers/main.py [hunk 2]
```diff
@@ -45,27 +46,55 @@ def shop(self, page=0, category=None, search='', ppg=False, **post):
...
+                                                  reverse=True),
+                'hottest':
+                    filtered_products.sorted(key=lambda r: r.create_date),
+                'rating': filtered_products.sorted(key=lambda r: r.rating),
+                'popularity': filtered_products.sorted(key=lambda r: r.views)}
+            if post.get('product_sorter', '0') != '0':
+                sortby = post['product_sorter']
+                res.qcontext['sortby'] = sortby
+                ordered_products = keys.get(sortby)
+            elif request.httprequest.cookies.get('default_sort', 'False') != 'False':  # noqa
```
Comment (moylop260) [inline website_product_filters/controllers/main.py:89]: `# noqa?`

Comment (oscarolar) [inline website_product_filters/controllers/main.py:89]: Just because I didn't wanted to split the line for a few characters longer

**Comment by moylop260** [comment]
:+1:

**Comment by moylop260** [comment]
We will merge it
After, @hugho-ad will creates change to IMP performance.

**Pull request merged**